src/main/java/picard/illumina/NewIlluminaBasecallsConverter.java

```diff
@@ -69,12 +71,14 @@ public NewIlluminaBasecallsConverter(final File basecallsDir, final File barcode
                                          final SortingCollection.Codec<CLUSTER_OUTPUT_RECORD> codecPrototype,
                                          final Class<CLUSTER_OUTPUT_RECORD> outputRecordClass,
                                          final BclQualityEvaluationStrategy bclQualityEvaluationStrategy,
+                                         final boolean includeNonPfReads,
```

Comment:
If there is a good default value for this parameter, would it make sense to add another constructor that forwards to this one, for backward compatibility?